mdoc/src/main/scala/mdoc/modifier/ScastieModifier.scala

```diff
@@ -0,0 +1,82 @@
+package mdoc.modifier
```

Comment:
`mdoc.mods`? Anyway I'm sold on the plural (following https://softwareengineering.stackexchange.com/questions/75919/should-package-names-be-singular-or-plural/327653), but I'm not sold on the abbreviation, because you end up with `Modifier` for the class names, but `mod` for the package.

I would rather use `Mod` for the classes (`ScastieMod`), or use `modifiers` for the package name.

Reply:
I may eventually go with `Mod` instead of `Modifier` since it's quite long. I renamed the package to `mdoc.modifiers` for now since I agree with the stack exchange answer you linked about plural is nice when all members of the package have the same type.